cluster/examples/kubernetes/ceph/cluster-minimal.yaml

```diff
@@ -24,5 +24,8 @@ spec:
     allowMultiplePerNode: false
   dashboard:
     enabled: true
+  monitoring:
+    enabled: true  # requires Prometheus to be pre-installed
```

Comment:
In the example yamls we will need to set `enabled: false` so we can avoid the errors by default if prometheus isn't installed.

Reply:
done.